manifests/init.pp

```diff
@@ -245,9 +245,9 @@
 class patroni (
 
   # Global Settings
-  String $scope,
-  String $namespace = '/service/',
-  String $hostname = $facts['networking']['hostname'],
+  String[1] $scope,
+  String[1] $namespace = '/service/',
+  Stdlib::Fqdn $hostname = $facts['networking']['hostname'],
```

Comment:
I think hostname can be a shortname and not fully qualified.

Reply:
I checked that before I raised the PR, `Stdlib::Fqdn` accepts shortnames as well (and that's intentional, not a bug). In the  tests, `$hostname` is also set to just `foo` (that's the default coming from FacterDB).

Reply:
Seems this should stay as string according to the docs.


https://patroni.readthedocs.io/en/latest/patroni_configuration.html
```
name: PATRONI_NAME environment variable value if set, otherwise the current machine’s hostname.
```

Reply:
okay, updated it